frontend/src/screens/ProductListScreen.js

```diff
@@ -16,16 +23,30 @@ export default function ProductListScreen(props) {
     success: successCreate,
     product: createdProduct,
   } = productCreate;
+
+  const productDelete = useSelector((state) => state.productDelete);
+  const {
+    loading: loadingDelete,
```

Comment:
TypeError: Cannot destructure property 'loading' of 'menuDelete' as it is undefined.

Reply:
check your productDeleteReducer in productReducers.js and ensure that there's no typo on `return { loading: true/false }`...